kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/DefaultKroxyliciousTester.java

```diff
@@ -54,7 +60,9 @@ private KroxyliciousClients clients() {
     }
 
     private KroxyliciousClients clients(String virtualCluster) {
-        return new KroxyliciousClients(KroxyliciousConfigUtils.bootstrapServersFor(virtualCluster, kroxyliciousConfig));
+        KroxyliciousClients client = new KroxyliciousClients(KroxyliciousConfigUtils.bootstrapServersFor(virtualCluster, kroxyliciousConfig));
```

Comment:
should we check if it's in the `clients` map and return the existing one instead of instantiating a new KroxyliciousClients?

Reply:
The fact it puts the new one into the map over the top of the existing one is also a bit dodgy. Nothing would get to call `close()` on the old KafkaClients instance.